### DISCUSSION TITLE
anova.gllvm

### DISCUSSION THREAD

**Discussion by AlainZuur** [JenniNiku/gllvm#206] [General]
Hello Bert,

Feel free to ignore this comment....... anova.gllvm 

<img width="718" alt="image" src="https://github.com/user-attachments/assets/1ede8f5f-5ffc-493b-b216-2d4bd6133b37" />

M6 is the full model and M6.b is the sub-model without an interaction. M6 is the first in the command, yet anova.gllvm calls it Model 2. I would have thought that the first one you type in the anova.gllvm() function is called "Model 1" on the next line....just like the native anova function. But anova.gllvm does it the other way around. That may confuse some people. Especially because there is another 1 and 2 in the actual table (which doesn't match the order of Model 1 and Model 2).

Of course...you can argue that people should know what they are doing when they type anova.

Anyway....feel free to ignore.

Alain

**Comment by BertvanderVeen** [comment]
Thanks Alain, I have opened this as an issue with mark "enhancement"
